usaspending_api/references/models.py

```diff
@@ -383,6 +383,14 @@ class LegalEntity(DataSourceTrackedModel):
     small_business_description = models.TextField(blank=True, null=True)
     individual = models.CharField(max_length=1, blank=True, null=True)
 
+    def save(self, *args, **kwargs):
+        super(LegalEntity, self).save(*args, **kwargs)
+
+        try:
```

Comment:
Yeah, I could do it that way. This is one of those cases where you write it in a more complex way, then start paring things down as you debug, and then you get to this point, hah. I'll update it first thing.